### ISSUE TITLE
Build instructions, correction

### ISSUE THREAD

**Issue by RNavega** [armory3d/krom_blender#1]
Hi. Thank you for publishing all this source.
I want to try building it on my Windows 8.1 x64. Maybe I can help with writing some easy build instructions for novices like me.

I noticed this in the [Windows steps](https://github.com/armory3d/krom_blender#windows):
- mkdir lib
- cd lib
- svn checkout https://svn.blender.org/svnroot/bf-blender/trunk/lib/win64_vc14 lib/win64_vc14

That sequence would make a folder structure like this:

> (Some main folder)\
> --- blender\
> --- lib\lib\win64_vc14

But Blender's [Windows cmake](https://github.com/armory3d/krom_blender/blob/master/build_files/cmake/platform/platform_win32_msvc.cmake#L139) seems to expect this:
> (Some main folder)\
> --- blender\
> --- lib\win64_vc14\

i.e. just one lib folder, so the svn checkout is just towards a \win64_vc14\ folder, no additional \lib\\.
Anyway, I'll try to build it, I'll come back and say how it goes.

**Comment by RNavega** [comment]
So I managed to build it. It took a lot of effort.

I can run Blender, there is an "Armory" editor, but it doesn't do anything when I switch to it.
![editor](https://user-images.githubusercontent.com/28221053/35251654-0c3106f4-ffd4-11e7-9d9f-d5296ed788cd.png)

The addon is not installed, it's not clear how it should be done.
How can I proceed? I really want to try this engine.

**Comment by luboslenco** [comment]
Thanks, I edited the instructions.

This repo is blender 'only', I started a single-clone repo with the rest of the sdk at https://github.com/armory3d/armory_sdk, please give me a few more days so I can push Armory 0.2 and make it all work. I need to move some paths, and this can only by done on new builds to not break Armory for existing users updating through the built-in updater.

**Comment by AFWSI** [comment]
Hey Lubos,
I'm AFWS88 on armory forums. I finally was able to build. In the output in Visual Studio I got this when finished ========== Build: 150 succeeded, 0 failed, 0 up-to-date, 0 skipped ========== ,but there's still 3333 warnings listed. 
[compiling_errors.txt](https://github.com/armory3d/krom_blender/files/1668244/compiling_errors.txt)

**Comment by RNavega** [comment]
@luboslenco understood! I'll wait for the 0.2 push. Good luck.

**Comment by AFWSI** [comment]
Hey Lubos,

Once you get Armory 0.2 finished and released and work on the single clone repo sdk. Will we be able to build Armory ourselves just like releases? I manged to get the viewport working by trial and error ,but noticed several things doesn't work, like when you hit play in armory play you don't get the extra window with scene in it. I seen this thread https://github.com/armory3d/armory/issues/427 and I copied "latest Krom binaries" to armsdk/win32/Krom/win32 ,but that just makes a window flash really quick.
